md/hardware-and-software-requirements.md

```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 ## Software
 
 Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Portal).
-| | 7.0.x <br/> 7.1.x | 7.2.x <br/> 7.3.x  | 7.4.x | 7.5.x
-|:-|-|-|-|-
+| | Version
+|:-|-
 | **Operating system** |
-| Windows Server 2012 R2 64 bits | ✔ | ✔ | ✔ | ✔ |
-| Red Hat Enterprise Linux 6.5 64 bits | ✔ | ✔ |  ✔ | ✔ |
-| Ubuntu 14.04 LTS 64 bits | ✔ | ✔ |  ✔ | ✔ |
-| **Java Virtual Machine** | 
-| Oracle Java SE Runtime Environment 7 u67 | ✔ | ✔ | ✔ | |
-| Oracle Java SE Runtime Environment 8 u40 (see note 4) | ✔ | ✔ |  ✔ | ✔ |
-| OpenJDK 7u65 | ✔ | ✔ | ✔ | |
-| OpenJDK 8 | | | | ✔ |
+| Microsoft Windows Server | 2012 R2 64 bits and higher |
+| Red Hat Enterprise Linux |  6.5 64 bits and higher |
+| Ubuntu | 14.04 LTS 64 bits and higher |
+| **Java Virtual Machine** |
+| Oracle Java SE Runtime Environment | 8u112 (see note 3) |
+| OpenJDK | 8u112 (see note 3) |
 | **Application Server** |
-| Tomcat 7.0.55 | ✔ | | | |
-| Tomcat 7.0.67 |  | ✔ | ✔ | ✔ |
-| JBoss AS 7.1.1  |✔ | ✔ | | |
-| WildFly 10.1.0  | | | ✔ | ✔ |
-| Weblogic server 12c (12.1.\*) | ✔ | ✔ | ✔ | |
+| Apache Tomcat | 7.0.x (x > 67) |
+| Red Hat WildFly | 10.1.x |
 | **Database** |
-| MySQL 5.5.27 (see notes 1 and 2) | ✔ | ✔ | ✔ | ? |
-| PostgreSQL 9.3  |✔ | ✔ | ✔ | ✔ |
-| SQL Server 2012 (see note 3) | ✔ | ✔ | ✔ | ✔ |
-| Oracle 11gR2 (11.2.0.x) | ✔ | ✔ | ✔ | ✔ |
-| Oracle 12c (12.1.0.x.y) | | ✔ | ✔ | ✔ |
+| MySQL | 5.5.27 and higher (see note 1) |
+| PostgreSQL | 9.3 and higher |
+| SQL Server | 2012 (see note 2) |
+| Oracle  | 11gR2 (11.2.0.x) and 12c (12.1.0.x.y) |
 | **Browser** |
-| Internet Explorer 9.x and newer | ✔ | ✔ | ✔ | |
-| Firefox (2 last versions) | ✔ | ✔ | ✔ | ✔ |
-| Chrome (2 last versions) | ✔ | ✔ | ✔ | ✔ |
-| Microsoft Edge | | | | ✔ | 
+| Mozilla Firefox | latest version |
+| Google Chrome | latest version |
+| Microsoft Edge | latest version |
 | **Mobile** |
-| iOS 7.1 -- Safari -- Approved on Apple iPhone 5 | ✔ | ✔ | ✔ | ✔ |
-| Android 4.4 -- Chrome -- Approved on Nexus 5 | ✔ | ✔ | ✔ | ✔ |
+| Mozilla Firefox | latest version |
+| Google Chrome | latest version |
+| Apple Safari | latest version |
 
 Notes:
 
 1. MySQL must be configured with innoDB storage engine.
-2. **Warning (7.0.x only, fixed starting 7.1.0) :** There are known issues with the management of XA transactions by MySQL engine and driver: see MySQL bugs [17343](http://bugs.mysql.com/bug.php?id=17343) and [12161](http://bugs.mysql.com/bug.php?id=12161) for more details.
```

Comment:
Warning only apply to version 7.0.x. As page focus on 7.4.x removing the warning seems ok to me.